docker/tracker_dcs_stack/docker-compose.dummy.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+services:
+  hv:
+    image: ghcr.io/cdozen/dummy:main
```

Comment:
use a semver tag for the image, e.g. 1.0.1

Reply:
Done!